webwhatsapi/objects/message.py

```diff
@@ -42,8 +42,6 @@ def factory_message(js_obj, driver):
 
 class Message(WhatsappObject):
 
-    sender: Union[Contact, bool]
```

Comment:
I don't know, what does this line 
`sender: Union[Contact, bool]`
but correct syntax is bellow 
`sender: Union[Contact, bool]`